EDDI/MainWindow.xaml.cs

```diff
@@ -801,7 +801,7 @@ private void ConfigureSquadronFactionOptions(EDDIConfiguration configuration)
                 {
                     foreach (Faction faction in FactionSystem.factions)
                     {
-                        if (faction.Allegiance == configuration.SquadronAllegiance)
+                        if (faction.Allegiance != Superpower.None || faction.Allegiance == configuration.SquadronAllegiance)
```

Comment:
Is this supposed to be an 'or'? Just checking that we didn't mean to say 'and' here.

Reply:
The intent here is that if a power allegiance hasn't been defined, the user can choose from any faction in the system.